nix/cells/automation/pipelines.nix

```
@@ -39,10 +46,49 @@ in
     nomad.resources.cpu = 10000;
   };
 
+  benchmark = { config, ... }:
+    let
+      fact = config.actionRun.facts.${cloud.library.actions.benchmark.input}.value.github_body;
+      prNumber = toString fact.issue.number;
+
+      prRevision = lib.getExe (pkgs.writeScript "get-pr-rev" ''
+        curl \
+          -H "Accept: application/vnd.github+json" \
+          https://api.github.com/repos/input-output-hk/plutus/pulls/${prNumber} \
+          | jq -r .base.sha
+      '');
```

```suggestion
      prRevision = pkgs.writeScript "get-pr-rev" ''
        ${lib.getExe pkgs.curl} \
          -H "Accept: application/vnd.github+json" \
          https://api.github.com/repos/input-output-hk/plutus/pulls/${prNumber} \
          | ${lib.getExe pkgs.jq} -r .base.sha
      '';
```

- `pkgs.writeScript` builds the script at `$out`, hence `lib.getExe` is not necessary and actually breaks this; use `writeScriptBin` to get `$out/bin/get-pr-rev`
- interpolate full path to curl and jq in here so that they will be available at runtime